internal/e2e/cds_test.go

```diff
@@ -18,14 +18,14 @@ import (
 	"testing"
 	"time"
 
-	"github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
```

Comment:
goimports turds here and elsewhere

Reply:
Is there a way to disable this? I can turn off the imports, but it's nice to manage for me.

Reply:
I'm not sure, it just started to happen to me when I upgraded to vim-go 1.20. Maybe we need to file an issue upstream about goimports being silly.

Reply:
This is apparently by design: https://github.com/golang/go/issues/30051#issuecomment-461913682

I'll turn off goimports for now and fix up manually.